### PR TITLE
[DO-1235] Update FOP Converter

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: business_register/converter/fop_full.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            if not kved_name:
+                continue
+            kved = self.get_kved_from_DB(kved_code, kved_name)
+            if activity.xpath('PRIMARY'):
+                is_primary = activity.xpath('PRIMARY')[0].text == "так"
+            else:
+                is_primary = False
+            alredy_stored = False
+            if len(already_stored_foptokveds):
+                for stored_foptokved in already_stored_foptokveds:
```
Comment (criticus) [inline business_register/converter/fop_full.py:73]: 🍝

File: business_register/converter/fop_full.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+                    registration_date=registration_date,
+                    registration_info=registration_info,
+                    estate_manager=estate_manager,
+                    termination_date=termination_date,
+                    terminated_info=terminated_info,
+                    termination_cancel_info=termination_cancel_info,
+                    contact_info=contact_info,
+                    vp_dates=vp_dates,
+                    authority=authority,
+                    code=code)
```
Comment (roman-tiukh) [inline business_register/converter/fop_full.py:211]:
```suggestion
                    code=code
                )
```

**Review by roman-tiukh — APPROVED** [review]
This PR has several places where there are repetitive blocks of code that can be packed into universal blocks.

File: business_register/converter/fop_full.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/fop_full.py:74]: This is heredity :)

File: business_register/converter/fop_full.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/converter/fop_full.py]: fixed

File: business_register/converter/fop_full.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/fop_full.py:74]: This is heredity :)

**Pull request merged**